test/testpaint/Data.cpp

```diff
@@ -625,261 +625,261 @@ const_utf8string FlatTrackNames[256] = {
+    "TrackElemType::LeftQuarterTurn5TilesUp25",
+    "TrackElemType::RightQuarterTurn5TilesUp25",
+    "TrackElemType::LeftQuarterTurn5TilesDown25",
+    "TrackElemType::RightQuarterTurn5TilesDown25",
+    "TrackElemType::SBendLeft",
+    "TrackElemType::SBendRight",
+    "TrackElemType::LeftVerticalLoop",
+    "TrackElemType::RightVerticalLoop",
+    "TrackElemType::LeftQuarterTurn3Tiles",
+    "TrackElemType::RightQuarterTurn3Tiles",
+    "TrackElemType::LeftBankedQuarterTurn3Tiles",
+    "TrackElemType::RightBankedQuarterTurn3Tiles",
+    "TrackElemType::LeftQuarterTurn3TilesUp25",
+    "TrackElemType::RightQuarterTurn3TilesUp25",
+    "TrackElemType::LeftQuarterTurn3TilesDown25",
+    "TrackElemType::RightQuarterTurn3TilesDown25",
+    "TrackElemType::LeftQuarterTurn1Tile",
+    "TrackElemType::RightQuarterTurn1Tile",
+    "TrackElemType::LeftTwistDownToUp",
+    "TrackElemType::RightTwistDownToUp",
+    "TrackElemType::LeftTwistUpToDown",
+    "TrackElemType::RightTwistUpToDown",
+    "TrackElemType::HalfLoopUp",
+    "TrackElemType::HalfLoopDown",
+    "TrackElemType::LeftCorkscrewUp",
+    "TrackElemType::RightCorkscrewUp",
+    "TrackElemType::LeftCorkscrewDown",
+    "TrackElemType::RightCorkscrewDown",
+    "TrackElemType::FlatToUp60",
+    "TrackElemType::Up60ToFlat",
+    "TrackElemType::FlatToDown60",
+    "TrackElemType::Down60ToFlat",
+    "TrackElemType::TowerBase",
+    "TrackElemType::TowerSection",
+    "TrackElemType::FlatCovered",
+    "TrackElemType::Up25Covered",
+    "TrackElemType::Up60Covered",
+    "TrackElemType::FlatToUp25Covered",
+    "TrackElemType::Up25ToUp60Covered",
+    "TrackElemType::Up60ToUp25Covered",
+    "TrackElemType::Up25ToFlatCovered",
+    "TrackElemType::Down25Covered",
+    "TrackElemType::Down60Covered",
+    "TrackElemType::FlatToDown25Covered",
+    "TrackElemType::Down25ToDown60Covered",
+    "TrackElemType::Down60ToDown25Covered",
+    "TrackElemType::Down25ToFlatCovered",
+    "TrackElemType::LeftQuarterTurn5TilesCovered",
+    "TrackElemType::RightQuarterTurn5TilesCovered",
+    "TrackElemType::SBendLeftCovered",
+    "TrackElemType::SBendRightCovered",
+    "TrackElemType::LeftQuarterTurn3TilesCovered",
+    "TrackElemType::RightQuarterTurn3TilesCovered",
+    "TrackElemType::LeftHalfBankedHelixUpSmall",
+    "TrackElemType::RightHalfBankedHelixUpSmall",
+    "TrackElemType::LeftHalfBankedHelixDownSmall",
+    "TrackElemType::RightHalfBankedHelixDownSmall",
+    "TrackElemType::LeftHalfBankedHelixUpLarge",
+    "TrackElemType::RightHalfBankedHelixUpLarge",
+    "TrackElemType::LeftHalfBankedHelixDownLarge",
+    "TrackElemType::RightHalfBankedHelixDownLarge",
+    "TrackElemType::LeftQuarterTurn1TileUp60",
+    "TrackElemType::RightQuarterTurn1TileUp60",
+    "TrackElemType::LeftQuarterTurn1TileDown60",
+    "TrackElemType::RightQuarterTurn1TileDown60",
+    "TrackElemType::Brakes",
+    "TrackElemType::RotationControlToggle",
     "TRACK_ELEM_INVERTED_90_DEG_UP_TO_FLAT_QUARTER_LOOP",
```

Comment:
Seems like one was missed?

Reply:
Missed a few of these in testpaint, but should be sorted now.